[[introduction]]
== Introduction

=== Overview

GBIF.org offers access to extensive biodiversity data, mainly focusing on where and when species have been observed. These records, sourced from collections, surveys, and citizen science, include species names, dates, locations, and  additional details. GBIF also accomodates DNA-associated biodiversity data, including  DNA metabarcoding data, and the first version of a practical dna-publishing guide was published in 2021. During consultations with the GBIF network and the research community in 2022 and 2023, it became clear that tools for easier data formatting and publishing were needed for DNA metabarcoding data.

THE_TOOL allows users with basic knowledge of data standards, GBIF and DNA data to reshape traditional dataset formats of the DNA metabarcoding community – OTU table type datasets – into standardised datasets possible to publish to GBIF. 

=== Target audiences

This guide is designed for anyone interested in publishing DNA metabarcoding data on GBIF or similar biodiversity platforms, catering to both seasoned GBIF network members and researchers new to GBIF and the data publishing process.

All new users are encouraged to review and go through both quick start guides to familiarize themselves with THE_TOOL and the processing concepts, and then consult other relevant sections of the guide and ressources as needed.

=== Scope

* THE_TOOL handles *DNA metabarcoding datasets* in the shape an OTU tables/matrices and associated metadata. Datasets can be uploaded in a few selected template formats.
** THE_TOOL cannot handle or process raw sequencing data (fastq files).
* THE_TOOL does [.underline]#not# handle DNA metagenomic data (shotgun sequencing data).
* THE_TOOL does [.underline]#not# handle other types of DNA biodiversity data (specimen barcodes, qPCR)

=== Essential terms

There is a full glossary in the end of the document. The list here contains some essential terms for quick reference to new users. 

* https://dwc.tdwg.org/[Darwin Core^] (Abbr: DwC): The name of the data standard used by GBIF.
* Darwin Core [.underline]#term#: a standardised field name (e.g. term:dwc[decimalLatitude] is the official DwC term for geographical latitude).
* Occurrence: the GBIF and DwC word for a registration/detection of a species/taxon in space and time.
* Occurrence core: This is the part of DwC that includes all the central information (fields) on biological occurrences in GBIF (e.g. spatiotemporal data, taxonomy, etc), also for eDNA data.
* dna-derived data: An extension to Occurrence core to capture information relating to DNA (e.g. primers, the sequence, sequencing platform, etc.). This extension is based on the MIxS standard used by the "GenBanks". 
* OTU (Operational Taxonomic Unit): In this context it refers to the molecular units / molecular species detected in an eDNA metabarcoding study. OTUs are often a cluster of higly similar species represented by one selected sequence type.
* ASV: Amplicon sequence variant. A variant of OTUs where where only denoising of the sequence data, but no clustering, has been performed. For simplicity we refer to OTUs in general to cover also ASVs.
* OTU table: spreadsheet that holds the number of sequencing reads detected of each OTU/sequence in each sample. Here used broadly to also cover ASVs.
* Darwin Core archive (DwC-A): A standardized format for sharing biodiversity data – a ZIP file that contains one or more data files along with a descriptor file that explains how the data files are organized.
* Endpoint: In the context of GBIF, an "endpoint" refers to a URL or web address where a DwC-A can be accessed through the internet, and indexed by GBIF.