[[introduction]]
== Introduction

=== Overview

GBIF offers access to extensive biodiversity data, mainly focusing on where and when species have been observed. These records, sourced from collections, surveys, and citizen science, include species names, dates, locations, and  additional details. GBIF also accomodates DNA-associated biodiversity data, including  DNA metabarcoding data, and the first version of a https://doi.org/10.35035/doc-vf1a-nr22[dna-publishing guide] was published in 2021. During consultations with the GBIF network and the research community in 2022 and 2023, it became clear that tools for easier data formatting and publishing were needed for DNA metabarcoding data.

THE_TOOL allows users with basic knowledge of data standards, GBIF and DNA (metabarcoding) data to reshape datasets in formats familiar to the DNA metabarcoding community – OTU table type datasets – into standardized datasets possible to publish to GBIF.

=== Target audiences

This guide is for anyone interested in publishing DNA metabarcoding data on GBIF (or OBIS, etc.), catering to both seasoned GBIF network members and researchers new to GBIF and the data publishing process.

[.underline]#All new users# are encouraged to review and go through both quick start guides to familiarize themselves with THE_TOOL, and then consult other relevant sections of the guide and ressources as needed.

=== Scope

* THE_TOOL handles *DNA metabarcoding datasets* in the shape of OTU tables/matrices and associated metadata. Datasets can be uploaded in a few selected template formats.
+
NOTE: THE_TOOL cannot handle or process raw sequencing files (fastq) from such datasets.
* THE_TOOL does [.underline]#not# handle DNA metagenomic data (shotgun sequencing data).
* THE_TOOL does [.underline]#not# handle other types of DNA-associated biodiversity data (specimen barcodes, qPCR, etc).

=== Getting started



=== Essential terms

There is a full glossary at the end of the document. The list here contains some essential terms for quick reference to new users. 

https://dwc.tdwg.org/[Darwin Core^] (DwC):: The name of the data standard used by GBIF.
Darwin Core [.underline]#term#:: a standardized field name (e.g. term:dwc[decimalLatitude] is the official DwC term for geographical latitude).
Darwin Core Archive (DwC-A):: A standardized format for sharing biodiversity data – a ZIP archive that contains one or more data files along with a descriptor file that explains how the data files are organized.
Occurrence:: the GBIF and DwC word for a registration/detection of a species/taxon in space and time.
Occurrence core:: This is the part of DwC that includes all the central information (fields) on biological occurrences in GBIF (e.g. spatiotemporal data, taxonomy, etc), also for eDNA data.
dna-derived data:: An extension to Occurrence core to capture information relating to DNA (e.g. primers, the sequence, sequencing platform, etc.). This extension is based on the MIxS standard used by the "GenBanks". 
OTU (Operational Taxonomic Unit):: In this context it refers to the molecular units / molecular species (OTUs, ASVs, sequences) detected in an eDNA metabarcoding study. OTUs are represented by one selected sequence.
ASV:: Amplicon sequence variant. A variant of OTUs where where only denoising of the sequence data, but no clustering, has been performed.
OTU table:: spreadsheet that holds the number of sequencing reads detected of each OTU/sequence in each sample. Here used broadly to also cover ASVs.
Endpoint:: In the context of GBIF, an "endpoint" refers to a URL or web address where a DwC-A can be accessed through the internet, and indexed by GBIF.